data/publiccloud/terraform/azure.tf

```diff
@@ -0,0 +1,154 @@
+# Configure the Microsoft Azure Provider
+provider "azurerm" {}
+
+variable "count" {
+ default = "1"
+}
+
+variable "name" {
+ default = "openqa-vm"
+}
+
+variable "type" {
+ default = "Standard_A2_v2"
+}
+
+variable "region" {
+ default = "westeurope"
+}
+
+variable "image_id" {
+ default = "SLES12-SP4-Azure-BYOS.x86_64-0.9.0-Build3.51.vhd"
+}
+
+// Random plugin for names
+resource "random_id" "service" {
+ count = "${var.count}"
+ keepers {
+  name          = "${var.name}"
+ }
+ byte_length = 8
+}
+
+# Create a resource group if it doesn’t exist
```

Comment:
I am not sure that coments like this really make sense

Reply:
Right, I removed all the comments.